Source/ACE/Network/GameEvent/Events/GameEventPlayerDescription.cs

```diff
@@ -339,28 +319,33 @@ private void WriteEventBody()
             /*if ((optionFlags & DescriptionOptionFlag.Shortcut) != 0)
             {
             }*/
-
-            Writer.Write(0u);
-
-            /*if ((optionFlags & DescriptionOptionFlag.SpellTab) != 0)
+            if ((optionFlags & CharacterOptionDataFlag.SpellLists8) != 0)
             {
-                // additional spell tabs 2-7
-                writer.Write(0u);
-                writer.Write(0u);
-                writer.Write(0u);
-                writer.Write(0u);
-                writer.Write(0u);
-                writer.Write(0u);
-            }*/
+                for (int i = 0; i <= 7; i++)
+                {
+                    var sorted =
+                        aceObj.SpellsInSpellBars.FindAll(x => x.AceObjectId == aceObj.AceObjectId && x.SpellBarId == i)
```

Comment:
This is a special case and I think in this instance should be allowed.   The server never does anything with these.   It is only a persistence function.   It saves them and it reads them to load them.  Everything else happens client side in regards to spell bars.   I can easily add another layer - but it would be for nothing I believe.